src/app/features/tech-record/components/tech-record-generate-letter/tech-record-generate-letter.component.ts

```diff
@@ -64,11 +64,10 @@ export class GenerateLetterComponent {
   }
 
   get letter(): LettersOfAuth | undefined {
-    return this.currentTechRecord?.letterOfAuth ?? undefined;
+    return (this.currentTechRecord as any)?.letterOfAuth ?? undefined;
```

Comment:
Does letterOfAuth not exist on the V3TechRecordModel yet? If not, should it?

Reply:
So as we're dealing with LGV no it doesn't exist on the current types as they are imported. I believe letters are trailer specific.